src/components/Pagination.jsx

```
@@ -1,35 +1,74 @@
-        size="sm"
-        className={page === 1 ? 'cursor-not-allowed opacity-50' : ''}
-      />
-      <span className="text-sm font-medium">
-        Page {page} of {totalPages}
-      </span>
-      <Button
-        onlyIcon
-        icon={<ChevronRight size={18} />}
+        disabled={page === 1}
+        className={`text-cornflower-950 hover:bg-cornflower-100 hover:text-cornflower-600 inline-flex items-center justify-center rounded-xl p-2 transition-colors ${page === 1 ? 'cursor-not-allowed opacity-50' : ''}`}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)